### PR TITLE
fix(ui): toolbox items labels i18n

### LINKED ISSUE #2030: [Bug] 2.24.0 - Tool title is not translated anymore
Upgrading from 2.23.2 to 2.24.0 has a regression for tool title translations.

Screenshots:

Before: 

Screenshot 2022-04-26 at 23 19 30

After:

Screenshot 2022-04-26 at 23 19 45

### PR DESCRIPTION
**Pull request by TatianaFomina**
Return the ability to display localized labels for toolbox items

### CODE REVIEW

**Review by ilyamore88 — APPROVED** [review]
👍

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+### 2.24.1
+— `Fix` — Tool title is not translated anymore [#2030](https://github.com/codex-team/editor.js/issues/2030)
```
Comment (gohabereg) [inline docs/CHANGELOG.md]:
```suggestion

— `Fix` — Tool title is not translated anymore [#2030](https://github.com/codex-team/editor.js/issues/2030)
```

**Review by gohabereg — APPROVED** [review]

File: docs/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+### 2.24.1
+
+— `Fix` — Tool title is not translated anymore [#2030](https://github.com/codex-team/editor.js/issues/2030)
```
Comment (neSpecc) [inline docs/CHANGELOG.md]:
```suggestion
— `Fix` — The I18n of Tools` titles at the Toolbox now works correctly [#2030](https://github.com/codex-team/editor.js/issues/2030)
```

File: src/components/ui/toolbox.ts [hunk 3]
```diff
@@ -133,7 +135,7 @@ export default class Toolbox extends EventsDispatcher<ToolboxEvent> {
       items: this.toolsToBeDisplayed.map(tool => {
         return {
           icon: tool.toolbox.icon,
-          label: tool.toolbox.title,
+          label: I18n.t(I18nInternalNS.toolNames, tool.toolbox.title),
```
Comment (neSpecc) [inline src/components/ui/toolbox.ts]: what if user (or tool itself) does not specified the `toolbox.title`? The title is the optional field.

Please, test this case. Maybe there should be something like `tool.title || _.capitalize(tool.name)`

**Review by neSpecc — CHANGES_REQUESTED** [review]

**Comment by neSpecc** [comment]
what do you think about adding a tests for these cases?

**Comment by TatianaFomina** [comment]
> what do you think about adding a tests for these cases?

Added a test

File: src/components/ui/toolbox.ts [hunk 4]
```diff
@@ -133,7 +135,7 @@ export default class Toolbox extends EventsDispatcher<ToolboxEvent> {
       items: this.toolsToBeDisplayed.map(tool => {
         return {
           icon: tool.toolbox.icon,
-          label: tool.toolbox.title,
+          label: I18n.t(I18nInternalNS.toolNames, tool.toolbox.title || _.capitalize(tool.name)),
```
Comment (neSpecc) [inline src/components/ui/toolbox.ts:138]: >_.capitalize(tool.name)

this statement is not covered by tests

**Review by neSpecc — CHANGES_REQUESTED** [review]

File: src/components/ui/toolbox.ts (hunk 4 above)
Comment (TatianaFomina) [reply src/components/ui/toolbox.ts:138]: added a test case

**Review by neSpecc — APPROVED** [review]
Brilliant

**Pull request merged**